README.md

```diff
@@ -18,7 +18,12 @@ Install the necessary dependencies:
 
 Finally, still within the virtual environment, run:
 ```
-    beamer-agent
+    beamer-agent --keystore-file <keyfile> \
+                 --password <keyfile-password> \
+                 --l2a-rpc-url <source-l2-rpc-url> \
+                 --l2b-rpc-url <target-l2-rpc-url> \
+                 --deployment-dir <contract-deployment-dir> \
+                 --token-match-file <token-match-file>
```

Comment:
While you're at this file, can you add a short description in the beginning and add a link to the testnet frontend as well?

Reply:
Good idea, will do.